### ISSUE TITLE
false positive with duplicate labels

### ISSUE THREAD

**Issue by dylanb** [dequelabs/axe-core#611]
The `aria-label` attribute in the snippet below overrides the `<label>` and should not show up as a violation but should show up only as a best practice.

```
<div class="-oneX-row">
    <label for="oneX-username" class="-oneX-textfield__floatingLabel claims-login-label">User ID</label>
    <input name="IDToken1" type="text" aria-label="Enter user ID" id="oneX-username" class="claims-login-input -oneX-textfield__input claims-login-input -oneX-stacked--first" placeholder="">
</div>
```

**Comment by WilcoFiers** [comment]
I'm unable to reproduce this.

**Comment by dylanb** [comment]
must have been fixed
